Automattic/wp-calypso
# WX Medium Plan: Create 'Plans' Page

Related to https://github.com/Automattic/wp-calypso/issues/74126
~Depends on https://github.com/Automattic/wp-calypso/pull/74140~

## Proposed Changes

* Added logic to the man Plans component to identify sites using Woo Express plans
* Show a different page to Woo Express
* Hide the `My Plan | Plans` toggle for Woo Express
* Put these changes under the `plans/wooexpress-medium` feature flag

## Testing Instructions

* Get a site with the WX Medium plan
   * Start with a free site
   * Go to `/checkout/<site-slug>/wooexpress-medium-bundle-monthly`
   * Finish checkout
* Go to the Plans page (`/plans/<site-slug>`)
* With the feature flag enabled (`/plans/<site-slug>?flags=plans/wooexpress-medium`) you should see a WX placeholder
![image](https://user-images.githubusercontent.com/3801502/223401857-955388b1-656f-41b8-9415-cbfadd2876a0.png)

* With the feature flag disabled (`/plans/<site-slug>?flags=-plans/wooexpress-medium`) you should see the default Plans page.
![image](https://user-images.githubusercontent.com/3801502/223401741-2a65380c-4ce8-42e3-9cb1-9a922a32c08d.png)
